pandas/tests/reshape/test_pivot.py

```diff
@@ -1625,3 +1625,9 @@ def test_isleapyear_deprecate(self):
 
         with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
             assert isleapyear(2004)
+
+    def test_pivot_margins_name_unicode(self):
+        # issue #13292
+        frame = pd.DataFrame({'foo': [1, 2, 3]})
+        pd.pivot_table(frame, index=['foo'], aggfunc=len, margins=True,
```

Comment:
still like to compare this here

Reply:
I figured out how to construct the expected object correctly. Please have a look.